src/components/Vizmapper/VisualPropertyRender/Color.tsx

```diff
@@ -19,7 +19,171 @@ export function ColorPicker(props: {
 
   return (
     <Box>
-      <ChromePicker
+      <PhotoshopPicker
+        color={localColorValue}
+        onChange={(color: any) => {
+          setLocalColorValue(color.hex)
+          debouncedValueChange(color.hex)
+        }}
+      />
+    </Box>
+  )
+}
+
+export function ColorPickerCompact(props: {
+  currentValue: ColorType | null
+  onValueChange: (color: ColorType) => void
+}): React.ReactElement {
+  const { onValueChange, currentValue } = props
+  const debouncedValueChange = debounce(onValueChange, 200)
+
+  // use local state to appear instantaneous in the color picker,
+  // but the actual visual style model updates are debounced
+  const [localColorValue, setLocalColorValue] = React.useState<ColorType>(
+    currentValue ?? `#ffffff`,
+  )
+
+
+  const customColors = ['#ffffff', '#d3d2ff', '#d3d2ff', '#d3d2ff', '#ffcfd1', '#ffcfd1', '#ffcfd1', '#ffffcf', '#c5ffcf', '#c5ffcf', '#c5ffcf', '#c5ffcf',
```

Comment:
These should be defined in the colors model impl so that they can be reused in other places in the app that needs to deal with colors.